samples/react-native/package.json

```diff
@@ -28,24 +28,25 @@
     "@sentry/react-native": "6.1.0",
     "delay": "^6.0.0",
     "react": "18.3.1",
-    "react-native": "0.75.4",
-    "react-native-gesture-handler": "^2.18.1",
-    "react-native-reanimated": "3.15.0",
-    "react-native-safe-area-context": "4.10.5",
-    "react-native-screens": "3.34.0",
-    "react-native-svg": "^15.6.0",
+    "react-native": "0.76.0",
+    "react-native-gesture-handler": "^2.20.2",
+    "react-native-reanimated": "3.16.1",
+    "react-native-safe-area-context": "https://github.com/th3rdwave/react-native-safe-area-context.git#91c426216c18b10558319f8f02383392100b6740",
```

Comment:
[This commit](https://github.com/th3rdwave/react-native-safe-area-context/commit/91c426216c18b10558319f8f02383392100b6740) which has not been released yet is needed for 0.76.0

Reply:
Included in `react-native-safe-area-context` `4.12.0`